src/test/resources/ssl/root-ca-invalid.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This cert is set to expire tomorrow. After expiry I will run the tests again and take this PR out of draft.

I have added a different expired cert to the `truststore_invalid.jks` keystore and verified that `./gradlew sslTest --tests SSLTest.testHttpsWithTrustStoreContainingInvalidCertsNotInChain` fails when there is an expired cert in the keystore that is not in the chain of the node certificate.